Source/misdat.cpp

```diff
@@ -129,4 +129,6 @@ MisFileData misfiledata[] = {
 	// clang-format on
 };
 
+size_t misfilelength = sizeof(misfiledata);

```

Comment:
do we really need a new variable for that? :D

Reply:
Yes, as the size of `misfiledata` array is only known in `misdate.cpp`, if I use `sizeof(misfiledata)` in `missiles.cpp` directly it causes `Invalid application of 'sizeof' to an incomplete type 'dvl::MisFileData []'` error. Unless there's a better way to do this that I don't know about, happy to amend if so 🙂